### PR TITLE
Playground announcement

### PR DESCRIPTION
**Pull request by rachied**
- Added blog entry to announce playground
- Added button to index.html linking to playground

### CODE REVIEW

File: blog/2023-06-23-announcing-stryker-playground.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+Explaining the intricacies of mutation testing to a co-worker or friend can be challenging.
+We believe the best way to grasp a new concept is through hands-on practice.
+While we already offer a [quickstart guide](/docs/General/example/), installing all the prerequisites can be quite cumbersome.
+
+
+That is why we are happy to announce the [Stryker Playground](https://stryker-mutator.io/stryker-playground/), bringing the Stryker.NET experience to the web!
+
+<!-- truncate -->
+
```
Comment (richardwerkman) [inline blog/2023-06-23-announcing-stryker-playground.md:15]: I would add a screenshot here to show how the playground looks

File: blog/2023-06-23-announcing-stryker-playground.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+Compiling the submitted code into an in-memory assembly is made possible by Roslyn, the .NET Compiler Platform.
+And of course the playground also depends on Stryker.NET for the instrumentation.
+
+Thanks to the [NUnit Lite Runner](https://docs.nunit.org/articles/nunit/running-tests/NUnitLite-Runner.html) we're able to discover and execute tests contained inside in-memory assemblies.
+
+# 🔮 What's on the horizon?
+We're looking for ways to improve intellisense, which should make it easier to freely write code and discover available methods and variables.
+We would also like to introduce a diverse collection of templates, offering users challenges of varying difficulties.
+
+And last but not least, supporting StrykerJs & Stryker4s is also on the roadmap!
```
Comment (richardwerkman) [inline blog/2023-06-23-announcing-stryker-playground.md]: I'd end with a call to action to try the playground for themselves and create issues on the repo with ideas or bug reports

File: blog/authors.yml [hunk 3]
```diff
@@ -39,3 +39,9 @@ xander-vedder:
   title: Stryker Team
   url: https://github.com/xandervedder
   image_url: https://avatars.githubusercontent.com/u/34238405?v=4
+
+rachid-boukaiba:
+  name: Rachid Boukaiba
+  title: Stryker.NET Team
+  url: https://rach.id
```
Comment (richardwerkman) [inline blog/authors.yml]: The url doesn't really show anything. Maybe it's better to link to your github page? Unless you plan to add anything to your site ;)

**Review by richardwerkman — COMMENTED** [review]
I'd add some more links to the playground on the site. Maybe at the bottom of the introduction page. Or even add an extra button in the header!

**Comment by psfinaki** [comment]
That's awesome, that will be actually super helpful - I always recreate something like this when I either explain mutation testing to somebody or just need to quickly check things out :)

On that note - it would be cool to "download" this playground, to have in a dev container or smth along those lines.

Another idea - putting the Stryker version somewhere there.

**Comment by rouke-broersma** [comment]
@psfinaki it's blazor so I think theoretically you should be able to just 'save' it as a pwa?

**Comment by psfinaki** [comment]
@rouke-broersma well I definitely don't have much knowledge on that but I guess PWA is usually shown like this:
image
Which is not the case here.

**Comment by rouke-broersma** [comment]
> @rouke-broersma well I definitely don't have much knowledge on that but I guess PWA is usually shown like this: image Which is not the case here.

Mhm, not sure what we would have to do to make it a PWA. Imo it should be theoretically possible anyway, there's no server component it's just a github pages static site. Sounds like a good feature request.

**Comment by hugo-vrijswijk** [comment]
You can make the page a PWA by adding a `manifest.json` and linking to it with a meta tag on the page. If you run Lighthouse with the PWA option enabled it gives a lot of info about what you need to do. This also has some good info: https://web.dev/pwa-checklist/

Also, this feature is awesome!!!

**Comment by nicojs** [comment]
@hugo-vrijswijk any idea why the build is parsing the CHANGELOG.md file from the stryker-js repo 🤦‍♂️?

```
:33:08 PM: SyntaxError: /opt/build/repo/docs/stryker-js/CHANGELOG.md: Expected corresponding JSX closing tag for <script>. (1027:337)
3:33:08 PM:   1025 | <h3 {...{id:bug-fixes-65}}>{`Bug Fixes`}</h3>
3:33:08 PM:   1026 | <ul>
3:33:08 PM: > 1027 | <li parentName=ul><strong parentName=li>{`vue:`}</strong>{` only mutate Vue files with `}<script>{` blocks (`}<a parentName=li {...{href:https://github.com/stryker-mutator/stryker/issues/1540}}>{`#1540`}</a>{`) (`}<a parentName=li {...{href:https://github.com/stryker-mutator/stryker/commit/ee4d27c}}>{`ee4d27c`}</a>{`)`}</li>
3:33:08 PM:        |                                                                                                                                                                                                                                                                                                                                                  ^
3:33:08 PM:   1028 | </ul>
3:33:08 PM:   1029 | <h3 {...{id:features-40}}>{`Features`}</h3>
3:33:08 PM:   1030 | <ul>
3:33:08 PM: [ERROR] Client bundle compiled with errors therefore further build is impossible.
```

**Comment by hugo-vrijswijk** [comment]
@nicojs I think it has to do with the way netlify caches the build differently to GH actions. I usually retry with a clean cache and stop thinking about it

**Comment by rachied** [comment]
@hugo-vrijswijk @nicojs can one of you clear the netlify cache for me?

**Comment by nicojs** [comment]
Note: I pushed some small improvements just to restart the pipeline thoroughly 😅

**Comment by nicojs** [comment]
Also: added an announcement: "📢 try out the new [Stryker Playground right here in your browser!](https://stryker-mutator.io/stryker-playground/)"
Also: merged it, because it is awesome! Thanks @rachied

**Comment by nicojs** [comment]
It's live: https://stryker-mutator.io/ 🎉

**Pull request merged**